src/jade-schema/structs/xref.struct.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "xref",
+  "fields": [
+    {
+      "name": "db",
+      "datatype": "string",
+      "type": "required"
+    },
+    {
+      "name": "id",
+      "datatype": "string",
+      "type": "required"
+    }
+  ]
```

Comment:
what about the `sourceName` field?

Reply:
Case in point for schema drift... updated!